src/content/en/updates/2021/01/devtools.md

```diff
@@ -0,0 +1,443 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Debugging support for Trusted Types violations, capture node screenshot beyond viewport, new Trust Tokens tab for network requests and more.
+
+{# wf_updated_on: 2021-01-19 #}
+{# wf_published_on: 2021-01-19 #}
+{# wf_tags: chrome89, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Debugging support for Trusted Types violations, capture node screenshot beyond viewport, new Trust Tokens tab for network requests and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 89) {: .page-title }
+
+{% include "web/_shared/contributors/jecelynyeen.html" %}
+
+<<../../_shared/devtools-research.md>>
+
+## Debugging support for Trusted Types violations {: #trusted-types }
+### Breakpoint on Trusted Type violations {: #trusted-types-violations }
+
+You can now set breakpoints and catch exceptions on Trusted Type Violations in the **Sources** panel.
+
+[Trusted Types](https://github.com/w3c/webappsec-trusted-types) API helps you prevent DOM-based cross-site scripting vulnerabilities. Learn how to write, review and maintain applications free of DOM XSS vulnerabilities with Trusted Types [here](https://web.dev/trusted-types/).
+
+In the **Sources** panel, open the **debugger** sidebar pane. Expand the **CSP Violation Breakpoints** section and enable the **Trusted Type violations** checkbox to pause on the exceptions. Try it yourself with [this demo page](https://tt-enforced.glitch.me/).
+
+![Breakpoint on Trusted Type violations](/web/updates/images/2021/01/devtools/trusted-type-violations.png)
+
+Could you add a picture explaining that the new issue icon at the end of the line shows issues raised at that line? Clicking the icon should take the user to the issues tab.
+
```

Comment:
This is probably a leftover comment from document review.

Reply:
thanks for notify, updated!